src/main/java/slimeknights/tconstruct/library/json/LevelingValue.java

```diff
@@ -63,7 +68,7 @@ public static LevelingValue eachLevel(float eachLevel) {
     return new LevelingValue(0, eachLevel);
   }
 
-  /** Trait to mix into a builder using leveling values */
+    /** Trait to mix into a builder using leveling values */
```

Comment:
Unneeded indentation change, make sure you don't have an autoformatter unless its consistent with the project style (and even then, I'd probably recommend only format changed lines)